frontend/packages/knative-plugin/console-extensions.json

```diff
@@ -504,6 +531,36 @@
       "required": ["FLAG_CAMEL_KAMELETS"]
     }
   },
+  {
+    "type": "console.catalog/item-type",
+    "properties": {
+      "type": "EventSink",
+      "title": "%knative-plugin~Event Sinks%",
+      "catalogDescription": "%knative-plugin~Event sinks are objects that link to an event producer and an event sink or consumer. Cluster administrators can customize the content made available in the catalog.%",
+      "typeDescription": "%knative-plugin~**Event sinks** are objects that link to an event producer and an event sink or consumer.%",
+      "filters": [
+        {
+          "label": "%knative-plugin~Provider%",
+          "attribute": "provider"
+        }
+      ]
+    },
+    "flags": {
+      "required": ["KNATIVE_EVENTING"]
```

Comment:
> required

@sahil143 this is something we can take in separate PR and this will be a new feature that will enable us to show status in the sidebar heading as we have on the details page not specific to events sink.

cc @serenamarie125

Reply:
i don't think we need `"FLAG_CAMEL_KAMELETS"` on `console.catalog/item-type`  as Eventsink card we are showing on Add feature flagged on `KNATIVE_EVENTING`. And have added `"FLAG_CAMEL_KAMELETS"` as feature flag to `console.catalog/item-provider`